internal/discover/graphics.go

```diff
@@ -271,13 +274,33 @@ func newXorgDiscoverer(logger logger.Interface, driver *root.Driver, nvidiaCTKPa
 		lookup.NewFileLocator(
 			lookup.WithLogger(logger),
 			lookup.WithRoot(driver.Root),
-			lookup.WithSearchPaths(libRoot, "/usr/lib/x86_64-linux-gnu"),
+			lookup.WithSearchPaths([]string{
+				libRoot + "/nvidia/xorg",
+				libRoot + "/xorg/modules/extensions",
+				libRoot + "/xorg/modules/drivers",
+				"/usr/lib/xorg/modules/extensions",
+				"/usr/lib/xorg/modules/drivers",
```

Comment:
This looks like we're searching the following 2 paths: `[]string{"extensions", "drivers"}` in `[]string{"modules", "modules/updates"}` across the following "roots": `[]string{libRoot + "/xorg", "/usr/lib", "/usr/lib64", "/usr/X11R6"}`.

Does it make sense to construct this dynamically?

Reply:
Yes, it would.

Reply:
Feel free to push to my branch.

Reply:
@elezar How's the progress?

Reply:
Addressed this one